jobbergate-api/jobbergate_api/apps/applications/routers.py

```diff
@@ -98,16 +119,20 @@ async def applications_delete_upload(
     """
     Delete application tarball using an authenticated user token.
     """
+    logger.debug(f"Preparing to delete tarball for {application_id=}")
+
     select_query = applications_table.select().where(applications_table.c.id == application_id)
     raw_application = await database.fetch_one(select_query)
+
     if not raw_application:
-        raise HTTPException(
-            status_code=status.HTTP_404_NOT_FOUND,
-            detail=f"Application {application_id=} not found.",
-        )
+        message = f"Application {application_id=} not found."
+        logger.warning(message)
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=message)
+
     application = ApplicationResponse.parse_obj(raw_application)
 
     if not application.application_uploaded:
+        logger.warning(f"Trying to delete a applications that was not uploaded ({application_id=})")
```

Comment:
```suggestion
        logger.warning(f"Trying to delete an application that was not uploaded ({application_id=})")
```

Reply:
solved in [0b4f677](https://github.com/omnivector-solutions/jobbergate/pull/121/commits/0b4f677de19511164ead7888b0e8b72006ea43e1)